src/katsdpcontroller/dashboard.py

```diff
@@ -279,14 +279,14 @@ def _make_app(self, **dash_args):
                 Output("task-table", "data"),
                 Output("capture-block-table", "data"),
                 Output("batch-table", "data"),
-                Output("root", "data-dummy"),
-            ],  # Just to set loading state on root
+                Output("root", "data-dummy"),  # Just to set loading state on root
+            ],
```

Comment:
Hard to say whether this will in general change the semantic association of the comment, but pretty poor form by `black`!